packages/ui/src/organisms/OutOfStock/OutOfStock.test.tsx

```diff
@@ -0,0 +1,121 @@
+import { render, screen } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import { axe } from 'jest-axe'
+import React from 'react'
+
+import Button from '../../atoms/Button'
+import Input from '../../atoms/Input'
+import Label from '../../atoms/Label'
+import OutOfStock from './OutOfStock'
+
+const SimpleOutOfStock = () => (
+  <OutOfStock>
+    <OutOfStock.Title>
```

Comment:
Curious, do editors/VSCode suggest `Title` and `Message` when you use `OutOfStock.*`? That'd be cool.